src/openPr.js

```diff
@@ -134,6 +105,10 @@ module.exports = async function ({ context, inputs, packageVersion }) {
       },
       inputs
     )
+    if (response?.status !== 201) {
+      const errMessage = response?.message || 'PR creation failed'
```

Comment:
are we sure that this is where the message will appear in case of an error?

Reply:
yes, I tested this by simulating an error